### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by jaylovespotato**
Translation on DAO DAO UI in Korean.

### CODE REVIEW

File: packages/i18n/locales/ko/translation.json [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    "tierVotingWeightPrompt_tokenBased": "총 토큰 공급량 중, {{weight}}  만큼의 양이 해당 티어의 모든 회원에게 균등하게 분배됩니다. 다른 투표 가중치를 가진 회원을 추가하시겠습니까? 새로운 티어를 추가합니다.",
+    "tierVotingWeight_membershipBased": "회원별 투표 가중치",
+    "tierVotingWeight_tokenBased": "총 토큰 공급량 중 비율(%)",
+    "toValidator": "검증인에게",
+    "tokenAddress": "토큰 주소",
+    "tokenContractAddressTitle": "토큰 계약 주소",
+    "tokenDistributionTitle": "토큰 분배",
+    "tokenImage": "토큰 이미지",
+    "tokenInfo": "토큰 정보",
+    "unstakingPeriodDescription": "투표하기 위해, DAO의 회원들은 자신의 토큰을 DAO에 스테이킹해야 합니다. DAO를 탈퇴하거나 거버넌스 토큰으로 교환하려는 회원은 먼저 토큰을 언스테이킹해야 합니다. 이 설정은 DAO의 회원이 토큰을 언스테이킹한 후 해당 토큰을 활용하기 위해 대기해야 하는 시간을 설정합니다. 이 기간을 길게 설정할수록 토큰을 등록하는 사람들이 DAO 거버넌스에 더욱 참여하기를 원한다고 볼 수 있습니다.”,
```
Comment (NoahSaso) [inline packages/i18n/locales/ko/translation.json:181]: I think this needs to be replaced with a normal quote.

**Review by NoahSaso — CHANGES_REQUESTED** [review]
Awesome! Thanks so much for doing this.

Could you add `ko` to the `locales` array in `packages/i18n/next-i18next.config.js` on line 7? After that and the JSON quote fix, should be good to go :)

**Comment by NoahSaso** [comment]
> Awesome! Thanks so much for doing this.
> 
> Could you add `ko` to the `locales` array in `packages/i18n/next-i18next.config.js` on line 7? After that and the JSON quote fix, should be good to go :)

Oh just saw your other PR. Nevermind about the config file then.

**Review by jaylovespotato — COMMENTED** [review]
File changed( line 181 : Normal Quotes )

File: packages/i18n/locales/ko/translation.json (hunk 1 above)
Comment (jaylovespotato) [reply packages/i18n/locales/ko/translation.json]: I fixed !! Thank u :)

**Comment by jaylovespotato** [comment]
Sorry for the slow progress. Is everything resolved?

**Review by NoahSaso — APPROVED** [review]

**Comment by NoahSaso** [comment]
Thanks so much for doing this!! Ready to merge :)

**Comment by NoahSaso** [comment]
Oh shoot, the JSON needs formatting. I'll push a quick commit to fix that :)

**Pull request merged**